### ISSUE TITLE
EOFError: Compressed file ended before the end-of-stream marker was reached

### ISSUE THREAD

**Issue by j-dutfield** [allenfrostline/PokerHandsDataset#2]
Hi, I'm getting the error below when running extract.py.

By this point it has generated folders for the different game modes but has not created the JSON file.

I thought that my download of the orginal IRC dataset may have been corrupted but downloading it again did not fix the problem

```
Extracting /Users/jamesdutfield/Documents/IRCPoker/IRCdata/holdem.199910.tgz
Traceback (most recent call last):
  File "extract.py", line 424, in <module>
    file_groups = loop_tgz(tgz_extract_directory)
  File "extract.py", line 403, in loop_tgz
    tar.extractall(extract_dir)
  File "/Users/jamesdutfield/opt/anaconda3/lib/python3.7/tarfile.py", line 2002, in extractall
    numeric_owner=numeric_owner)
  File "/Users/jamesdutfield/opt/anaconda3/lib/python3.7/tarfile.py", line 2044, in extract
    numeric_owner=numeric_owner)
  File "/Users/jamesdutfield/opt/anaconda3/lib/python3.7/tarfile.py", line 2114, in _extract_member
    self.makefile(tarinfo, targetpath)
  File "/Users/jamesdutfield/opt/anaconda3/lib/python3.7/tarfile.py", line 2163, in makefile
    copyfileobj(source, target, tarinfo.size, ReadError, bufsize)
  File "/Users/jamesdutfield/opt/anaconda3/lib/python3.7/tarfile.py", line 247, in copyfileobj
    buf = src.read(bufsize)
  File "/Users/jamesdutfield/opt/anaconda3/lib/python3.7/gzip.py", line 276, in read
    return self._buffer.read(size)
  File "/Users/jamesdutfield/opt/anaconda3/lib/python3.7/_compression.py", line 68, in readinto
    data = self.read(len(byte_view))
  File "/Users/jamesdutfield/opt/anaconda3/lib/python3.7/gzip.py", line 482, in read
    raise EOFError("Compressed file ended before the "
EOFError: Compressed file ended before the end-of-stream marker was reached
```

Any help would be much appreciated. Thanks

**Comment by j-dutfield** [comment]
<img width="1165" alt="Screenshot 2020-05-21 at 19 24 30" src="https://user-images.githubusercontent.com/60622327/82592488-ea82db00-9b98-11ea-85c9-d8d7d5e12838.png">

**Comment by hack-r** [comment]
this is a thing, but you can bypass it by modifying his try/catch such that if an EOFError occurs during the extraction of a specific .tgz file, it will print the error message and move on to the next file without stopping the entire process.

**Comment by wsdqy1234** [comment]
I have the same Question too

**Comment by allenfrostline** [comment]
fixed in 363a2be
